src/analysis/clustering.jl

```diff
@@ -199,3 +239,107 @@ function target_clusters(
     # Return new clusters vector with only 1 and 0 for target and non-target clusters
     return [c ∈ target_indexes ? 1 : 0 for c in clusters]
 end
+
+"""
+    function find_scenarios(outcomes::AbstractArray, clusters::AbstractMatrix, filter_functions::Vector{Function}; aggregation_function::Function=temporal_variability)::BitVector
+    function find_scenarios(outcomes::AbstractArray, clusters::AbstractMatrix, filter_function::Function; aggregation_function::Function=temporal_variability)::BitVector
+
+Find scenarios across a list of given metrics outcomes using clustering strategy:
+For each metric outcome:
+    - A median series is computed for each previously computed cluster (passed as argument)
+    - A summary statistics is computed for each median series (default is temporal variance)
+    - Selected clusters are those whose summary statistics is true for given function
+Selected scenarios are, then, those that belong to the selected cluster for all outcomes
+

```

Comment:
```suggestion
    - Selected clusters are those whose summary statistics is true for given function
    
Selected scenarios are, then, those that belong to the selected cluster for all outcomes.

```

I prefer a new line after dot points.